### ISSUE TITLE
Return key inserts line above current line after autocompletion using down arrow key

### ISSUE THREAD

**Issue by MaxGyver83** [kovisoft/paredit#51]
How to reproduce:

1. Create a new lisp file: `vim test.lisp`
2. Insert `(print)` in the first line.
3. Go to the second line and insert `(pr`.
4. Now (still in insert mode) start completion with `Ctrl-x Ctrl-n` (or just `Ctrl-n`).
5. Select `print` using the `down` key.
6. Press `enter`.

Then you get:

```lisp
(print)

(print)
```

An empty line was inserted above the second `(print)` form (and you are still in insert mode with the cursor in the second line).

![vim completion paredit](https://github.com/kovisoft/paredit/assets/13970628/686f989d-759e-4827-b602-d6106b1665e2)

This does not happen when I select the completion (`print`) using `Ctrl-n` or when I ignore the completion and press `enter` right away.

Same problem with commit `3609c63` (=before my last pull request), `master` and my branch `backtick-strings`.

At first I thought it was a problem with Janet files. But then I removed `janet.vim`, `conjure`, `equalprg=janet-format`, `vim-mucomplete`. All this made no difference. But now I know that this happens with `.lisp` files as well.

Any idea why this happens?

**Comment by MaxGyver83** [comment]
This issue seems to be present only with `let g:paredit_electric_return = 1`!

**Comment by MaxGyver83** [comment]
I think this is the cleanest way to reproduce the issue:

```sh
mkdir -p ~/.vim/pack/plugins/opt
cd ~/.vim/pack/plugins/opt
git clone https://github.com/kovisoft/paredit.git
[ -e ~/.vim/after/ftplugin/lisp.vim ] && mv ~/.vim/after/ftplugin/lisp.vim ~/.vim/after/ftplugin/bak_lisp.vim
echo '(print)' > /tmp/test.lisp
vim -u DEFAULTS -c 'set completeopt=menuone,noselect' -c 'pa paredit' -c 'edit' /tmp/test.lisp
```

Type:

```vim
o(pr<C-n><down><CR>
```

You end up with:

```lisp
(print)

(print)
```

Expected:

```lisp
(print)
(print

  )
```

Or maybe:

```lisp
(print)
(print)
```

**Comment by MaxGyver83** [comment]
Fixed in 989d1c046bbe36f7d71878636dbe6af8ea3f4f86.
